public/form.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Happy Chores</title>
+    <link rel="stylesheet" href="style.css">
+<!--    <script src="app.js"></script>-->
+</head>
+<body>
+<form name="interestForm" onsubmit="return formValidation()">
+    <label for='fname' >First Name</label><br>
+    <input type="text" size=65 id="fname" name="fname" placeholder="Enter your first name"/><br>
+    <label for="lname">Last Name</label><br>
+    <input type="text" size=65 id="lname" name="lname" placeholder="Enter your last name"/><br>
+    <label for="email" >Email Address*</label><br>
+    <input type="text" size=65 id="email" name="email" placeholder="Enter your email address" onkeyup="manage(this)"/><br>
+    <input type="submit" id="submit" disabled/>
+    <button type="clear" id="clear" name="clear" onclick="clearFields();"/>Clear</button><br>
+    <input type="checkbox" id="accept" onchange="manage(this)"/>
+    <label for="accept">By checking this box you accept the terms of this page.*</label>
+    <p>* Required Field</p>
+</form>
+</body>
+
+
+
+<script>
+function formValidation() {
+    const email = document.forms['interestForm']['email'];
+
+    if (email.value === '') {
+        window.alert("Cannot leave blank. Please enter an email address.");
+        return false;
+    }
+    return true;
+}
+
+function manage() {
+    const suButton = document.getElementById('submit');
+    const checkbox = document.getElementById('accept');
+
+    if (email.value != '' && checkbox.checked === true ) {
+        suButton.disabled = false;
+    } else {
+        suButton.disabled = true;
+        }
+    }
+
+function clearFields() {
```

Comment:
This works but every time you add a field to this form you will need to modify this function. Can you think of a better way?